solidity/test/normal/PCV.test.ts

```diff
@@ -584,41 +585,73 @@ describe("PCV", () => {
     })
 
     it("uses all fees to pay down the debt if feeSplitPercentage is 0, even if the feeRecipient is set and there is an active protocol bootstrap loan", async () => {
```

Comment:
All those tests with *and there is an active protocol bootstrap loan* part could use a rename. It no longer matters if there is or there is no active bootstrap loan, as we removed the `onlyAfterDebtPaid` modifier and the ` require((debtToPay > 0 && _feeSplitPercentage <= FEE_SPLIT_MAX) || (debtToPay == 0 && _feeSplitPercentage <= 100), "PCV: Fee split must be at most 50 while debt remains.");` restriction.